src/components/scenes/WalletDetailsScene.tsx

```diff
@@ -174,7 +175,12 @@ function WalletDetailsComponent(props: Props) {
   })
 
   const handlePressSeeAll = useHandler(() => {
-    navigation.navigate('transactionList', route.params)
+    navigation.navigate('transactionList', {
+      walletId: wallet.id,
+      tokenId,
+      walletName: getWalletName(wallet),
```

Comment:
Actually, I see we have the same bad pattern here on the TransactionList. Please export a `TransactionListTitle` component like we did for the `WalletDetailsTitle`:

```ts
export const TransactionListTitle = () => {
  const route = useRoute<RouteProp<'walletDetails'>>()
  const account = useSelector(state => state.core.account)
  const wallet = account.currencyWallets[walletId]
  const title = wallet == null ? '' :  getWalletName(wallet)
  return <HeaderTitle title={title} />
}
```